js/ui/map.js

```diff
@@ -287,6 +287,27 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
     },
 
     /**
+     * Set constraint on the map's geographical bounds. Pan or zoom operations that would result in displaying regions that fall outside of the bounds instead result in displaying the map at the closest point and/or zoom level of the requested operation that is within the max bounds.
+     *
+     * @param {LngLatBounds | Array<Array<number>> | null | undefined} lnglatbounds optional Desired max bounds of the map. If null or undefined, function removes any bounds constraints on the map.
+     * @returns {Map} `this`
+     */
```

Comment:
:+1: And also removing the word "optional", to further clarify that the intent is to explicitly pass `null` or `undefined`.